packages/core/src/workflows/workflow.ts

```diff
@@ -688,14 +705,6 @@ export class Workflow<
     return null;
   }
 
-  watch(onTransition: (state: WorkflowRunState) => void): () => void {
-    this.#onStepTransition.add(onTransition);
-
-    return () => {
-      this.#onStepTransition.delete(onTransition);
-    };
-  }
-
   async resume({
```

Comment:
The old resume function stays, but it didn't update to use the new one under the hood?